src/libPMacc/include/algorithms/math/floatMath/trigo.tpp

```diff
@@ -83,7 +84,7 @@ struct Sinc<float>
 {
     typedef float result;
 
-    HDINLINE double operator( )(const float& value )
```

Comment:
huh, rly looks like a very *minor* bug to me (promoted, not down-graded).
but can stay in the pull if you like.